python/pyspark/sql/group.py

```diff
@@ -192,7 +193,84 @@ def pivot(self, pivot_col, values=None):
             jgd = self._jgd.pivot(pivot_col)
         else:
             jgd = self._jgd.pivot(pivot_col, values)
-        return GroupedData(jgd, self.sql_ctx)
+        return GroupedData(jgd, self._df)
+
+    @since(2.3)
+    def apply(self, udf):
+        """
+        Maps each group of the current :class:`DataFrame` using a pandas udf and returns the result
+        as a `DataFrame`.
```

Comment:
"pandas udf" to "pandas_udf"

`DataFrame` to :class:`DataFrame`

Reply:
Fixed.

Reply:
I think "pandas udf"  as a word is fine. `pandas_udf` is the function name.